### PR TITLE
update for android 13

### PR DESCRIPTION
**Pull request by walnut-co**
Please double check and merge if possible.

### CODE REVIEW

File: sample/Xamarin.Forms.VonageSample/StartPage.cs [hunk 1]
```diff
@@ -15,6 +15,9 @@ public StartPage()
                         VerticalOptions = LayoutOptions.CenterAndExpand,
                         Text = "CLICK TO CHAT",
                         Command = new Command(() => {
+
+                            var permissionResult= CrossVonage.Current.CheckPermissions();
+
```
Comment (AndreiMisiukevich) [inline sample/Xamarin.Forms.VonageSample/StartPage.cs]:
```suggestion
```

File: lib/Xamarin.Forms.Vonage/VonagePermission.cs [hunk 2]
```diff
@@ -34,10 +35,14 @@ public enum VonagePermission
         /// The default audio device supports Bluetooth audio. If your app does not use the default audio device and does not use Bluetooth, you can remove this permission.
         /// </summary>
         Bluetooth,
+        ReadMediaAudio,
+        ReadMediaImages,
+        ReadMediaVideo,
```
Comment (AndreiMisiukevich) [inline lib/Xamarin.Forms.Vonage/VonagePermission.cs:40]: Please add doc comments

File: lib/Xamarin.Forms.Vonage.Android/Xamarin.Forms.Vonage.Android.csproj [hunk 3]
```diff
@@ -65,19 +67,17 @@
     <None Include="Resources\AboutResources.txt" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Xamarin.Android.Support.v7.AppCompat" Version="28.0.0.3" />
+    <PackageReference Include="Xamarin.AndroidX.AppCompat" Version="1.0.2-preview01" />
     <PackageReference Include="Xamarin.Forms">
-      <Version>5.0.0.2515</Version>
+      <Version>5.0.0.2578</Version>
```
Comment (AndreiMisiukevich) [inline lib/Xamarin.Forms.Vonage.Android/Xamarin.Forms.Vonage.Android.csproj:75]: if we are updating XF version here, let's update it in the entire solution (all projects)

File: lib/Xamarin.Forms.Vonage.Android/Services/PlatformVonageService.cs [hunk 4]
```diff
@@ -193,7 +194,7 @@ private IEnumerable<string> GetPermissions()
                 yield return Manifest.Permission.Camera;
             }
 
-            if (Permissions.HasFlag(VonagePermission.WriteExternalStorage))
+            if (Permissions.HasFlag(VonagePermission.WriteExternalStorage) && (int)Build.VERSION.SdkInt < 33)
```
Comment (AndreiMisiukevich) [inline lib/Xamarin.Forms.Vonage.Android/Services/PlatformVonageService.cs]: For consistency purpose

And let's put it right before `if ((int)Build.VERSION.SdkInt >= 33)` check

```csharp
if ((int)Build.VERSION.SdkInt < 33)
{
    if (Permissions.HasFlag(VonagePermission.WriteExternalStorage)
    {
        yield return Manifest.Permission.WriteExternalStorage;
    }
}
```

**Comment by walnut-co** [comment]
please check again and let me know if all ok.

**Review by AndreiMisiukevich — APPROVED** [review]

**Pull request merged**